locale/en.po

```diff
@@ -205,7 +205,8 @@ msgstr ""
 "Vacuole: Stores 8 collected compounds\n"
 "\n"
 "Toxin Vacuoles: Produces toxins (called [thrive:compound type=\"oxytoxy\"]"
-"[/thrive:compound])"
```

Comment:
You could use "proportional" here. It would be more precise, but it might end up being confusing and jargony. I'm fine with it either way.

Reply:
I changed the wording here a bit while I fixed typos I noticed in this PR ("ammount" was misspelled twice).